components/actionbutton/metadata/mods.md

```diff
@@ -47,4 +47,8 @@
 | `--mod-actionbutton-icon-size`                                    |
 | `--mod-actionbutton-label-color`                                  |
 | `--mod-actionbutton-min-width`                                    |
+| `--mod-actionbutton-static-content-color`                         |
 | `--mod-actionbutton-text-to-visual`                               |
+| `--mod-animation-duration-100`                                    |
+| `--mod-line-height-100`                                           |
+| `--mod-sans-font-family-stack`                                    |
```

Comment:
Yeah, I didn't want to mess too much with the script because I knew Josh's was going to merge soon but I needed to update it enough to run on individual packages so it could be a part of the build flow.